### PR TITLE
Remove support for Sorayomi web interface

### LINKED ISSUE #392: [Bug] Remove support for Sorayomi web interface
Seems to be an unused feature and is unmaintained.

### PR DESCRIPTION
**Pull request by marcoebbinghaus**
fixes #392

### CODE REVIEW

**Comment by marcoebbinghaus** [comment]
Please let me know if I missed something / if the ticket https://github.com/Suwayomi/Tachidesk-Server/issues/392 was meant to have a larger scope.

File: README.md [hunk 1]
```diff
@@ -48,8 +48,7 @@ Here's a list of known clients/user interfaces for Tachidesk-Server:
 ##### Actively Developed Cients
 - [Tachidesk-WebUI](https://github.com/Suwayomi/Tachidesk-WebUI): The web/ElectronJS front-end that Tachidesk-Server is traditionally shipped with. Usually gets new features faster.
 - [Tachidesk-JUI](https://github.com/Suwayomi/Tachidesk-JUI): The native desktop front-end for Tachidesk-Server. Currently the most advanced.
-- [Tachidesk-qtui](https://github.com/Suwayomi/Tachidesk-qtui): A C++/Qt front-end for mobile devices(Android/linux), in super early stage of development. 
-- [Tachidesk-Sorayomi](https://github.com/Suwayomi/Tachidesk-Sorayomi): A Flutter front-end for Desktop(Linux, windows, etc.), Web and Android. UI and UX similar to Tachiyomi.
```
Comment (AriaMoradi) [inline README.md]: Doesn't need to be removed

**Review by AriaMoradi — CHANGES_REQUESTED** [review]
I didn't look at the rest of the code, You basically just need to reverse #342

**Comment by marcoebbinghaus** [comment]
> I didn't look at the rest of the code, You basically just need to reverse #342

I looked through the changes from #342 and imo my commit reverses the relevant changes from there. I also undid the change in README.md as you commented.

Let me know if further alignments are needed.

**Review by Syer10 — APPROVED** [review]

**Pull request merged**